### PR TITLE
Html2canvas bugfix

### PR DESCRIPTION
**Pull request by rafalpospiech**
fixed printing html content

### CODE REVIEW

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js [hunk 1]
```diff
@@ -1283,7 +1283,7 @@ jQuery.Class('Vtiger_Widget_Js', {
 	 * @param {jQuery} element
 	 */
 	printHtml(element) {
-		let widget = element.closest('.dashboardWidget'),
+		let widget = $(element.closest('.dashboardWidget')),
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: ?

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js [hunk 2]
```diff
@@ -1307,9 +1307,9 @@ jQuery.Class('Vtiger_Widget_Js', {
 	 * @param {jQuery} element
 	 */
 	downloadHtmlAsImage(element) {
-		let widget = element.closest('.dashboardWidget'),
+		let widget = $(element.closest('.dashboardWidget')),
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: ?

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js [hunk 3]
```diff
@@ -1386,24 +1386,27 @@ jQuery.Class('Vtiger_Widget_Js', {
...
 	},
+	printImage(imgEl, title, width, height) {
+		const print = window.open('', 'PRINT', 'height=' + height + ',width=' + width);
+		print.document.write('<html><head><title>' + title + '</title>');
+		print.document.write('</head><body >');
+		print.document.write($('<div>').append(imgEl).html());
+		print.document.write('</body></html>');
+		print.document.close(); // necessary for IE >= 10
+		print.focus(); // necessary for IE >= 10
+		setTimeout(function () {
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: ?

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js [hunk 4]
```diff
@@ -1386,24 +1386,27 @@ jQuery.Class('Vtiger_Widget_Js', {
...
-			print.document.write('</head><body >');
-			print.document.write($('<div>').append(imgEl).html());
-			print.document.write('</body></html>');
-			print.document.close(); // necessary for IE >= 10
-			print.focus(); // necessary for IE >= 10
-			setTimeout(function () {
-				print.print();
-				print.close();
-			}, 1000);
+			this.printImage(imgEl, header.find('.dashboardTitle').text(), 600, 400);
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: 600 >100%?

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js [hunk 5]
```diff
@@ -1296,7 +1296,7 @@ jQuery.Class('Vtiger_Widget_Js', {
 			if (height < 400) {
 				height = 400;
 			}
-			this.print(imgEl.get(0), title, width, height);
+			this.printImage(imgEl.get(0), title, width, height);
```
Comment (rafalpospiech) [inline public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: print image is used in two places (DRY) , before there was this.print method but I have no idea where it sink

File: public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js (hunk 4 above)
Comment (rafalpospiech) [reply public_html/layouts/basic/modules/Vtiger/resources/dashboards/Widget.js]: get screen resolution and open window in full screen?

**Pull request merged**